relayer/cmd/run/execution/command.go

```diff
@@ -0,0 +1,104 @@
+package execution
+
+import (
+	"context"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/sirupsen/logrus"
+	"github.com/snowfork/snowbridge/relayer/chain/parachain"
+	"github.com/snowfork/snowbridge/relayer/relays/execution"
+	"github.com/snowfork/snowbridge/relayer/relays/execution/config"
```

Comment:
I agree the package structure should be consistent. However, I added the config package to avoid a cyclical import in `message/message.go` (using `config.go` in the parent package produces a cyclical import). This is kind of related to the [question here](https://github.com/Snowfork/snowbridge/pull/721#discussion_r1022712303). See comment there too.

I think using consistent packages for the parachain and beefy relayers would be great, but not in the scope of this PR. I'll make a follow-up.